errata/ko/errata.json

```diff
@@ -455,7 +455,7 @@
       "campaign_code": "rttcu",
       "questions": [
         {
-          "question": "[돌아온 황혼회 저택에서의 실종] 조사자들의 게임 준비를 진행하고자 합니다만, 게임 준비에 언급된 카드의 일부가 대체되었습니다. 이 경우 어떻게 게임 준비를 수행하여야 하나요?",
```

Comment:
Oh I added support for this card to my latest release, you don't need to remove this.

Reply:
This is not removing the entire, just partially deleting the phrase/card that indicates it is RTTCU ONLY.